tests/unit/test_dynamodb.py

```diff
@@ -0,0 +1,116 @@
+import os
+
+import pytest
+from moto import mock_aws
+
+from polytope_server.common import request, user
+from polytope_server.common.request_store import dynamodb_request_store
+
+
+@pytest.fixture(scope="function")
+def aws_credentials():
```

Comment:
These variables are not cleaned up after testing, depending on the environment could persist.

Reply:
good point